riscv-sbi.adoc

```diff
@@ -257,6 +290,291 @@ call doesn't return.
 | *RESERVED*                |             |    0x09-0x0F |
 |===
 
+
+== Timer Extension, Extension ID: 0x54494D45 (TIME)
```

Comment:
I'm not sure purposefully leaving the important extensions sparsely mapped is a good idea.  IDK how much performance is going to be left on the table here, it just seems needless.

Reply:
Currently, there are no performance impact as we are doing a direct comparison. It was kept sparse to make IDs assignment more logical (ASCII text). But I don't strongly feel about this. So I am happy to modify them to continuous if you prefer.